### PR TITLE
Design/user logo change

### PR DESCRIPTION
**Pull request by taisuou**
changed user logo from Yahoo Japan's to LINE corporation's.

change-logo-white
change-logo-black

### CODE REVIEW

**Comment by kpango** [comment]
how about using lycorp en logo.

**Comment by vankichi** [comment]
Unfortunately, I think there is no English ver. logo.

File: themes/vald/layouts/_default/index.html [hunk 1]
```diff
@@ -475,12 +475,22 @@ <h2 class="about__title">SDKs</h2>
       <h2 class="about__title">Vald Users</h2>
       <ul class="about__list about__list--center">
         <li class="about__item about__item--single">
-          <a href="https://yahoo.co.jp/" target="_blank"
-            ><img
-              class="about__logo"
-              src="{{ .Site.BaseURL }}/images/yahoojapan.svg"
-              alt="yahoojapan"
-          /></a>
+          <a href="https://www.lycorp.co.jp/" target="_blank">
```
Comment (kpango) [inline themes/vald/layouts/_default/index.html]:
```suggestion
          <a href="https://www.lycorp.co.jp/en/" target="_blank">
```

**Review by kpango — APPROVED** [review]
LGTM

**Review by vankichi — APPROVED** [review]

**Pull request merged**